app/services/boarding_service.rb

```diff
@@ -74,9 +74,13 @@ def ensure_values
       spaceship = Spaceship::Tunes.login(@user, @password)
       spaceship.select_team
 
-      @app ||= Spaceship::Tunes::Application.find(@app_id)
-      error_message << "Could not find app with ID #{app_id}" if @app == nil
-
+      @app ||= Spaceship::Tunes::Application.find(@app_id)      
+      if @app.nil?
```

Comment:
then you could change this to something like `raise "Could not find app with ID #{app_id}"` if @app.nil?